src/components/logs/forms/medication-setup.tsx

```
@@ -0,0 +1,217 @@
+        defaultValues: {
+            name: "",
+            medicationForm: "",
+            strength: "",
+            defaultDoseUnits: 0,
+            notes: ""
+        },
+        validators: {
+            onChange: medicationSetupSchema
+        },
+        onSubmit: ({ value }) => {
+            if (!medicationForms.includes(value.medicationForm as any)) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)